imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/TIFFImageReader.java

```diff
@@ -830,7 +832,7 @@ else if (bitsPerSample == 64) {
 
                 throw new IIOException("Unsupported BitsPerSample for SampleFormat 3/Floating Point (expected 16/32/64): " + bitsPerSample);
```

Comment:
Same for this message, use the actual sample format in the message, not just hardcoded 3 (FP).